### DISCUSSION TITLE
Orient Me throws HTTP Error 500 after Quick Start Guide

### DISCUSSION THREAD

**Discussion by jmosen** [HCL-TECH-SOFTWARE/connections-automation#100] [Q&A] [answered]
Hi,
I followed the quick start guide for CNX7 + Component Pack and applied the connections-post-install playbook. Everything works as expected, but now web.example.org/social throws HTTP error 500 for Orient Me. 
I already looked up the logs for orient-web-client in Kubernetes and it says:
`{"pid":7,"hostname":"orient-web-client-7f76d5ff64-cvgv2","name":"@connections/orient-web-client/@ibm-ics/common-lib-gatekeeper/dist8/src/persistance/redis/redis-utils.js","level":50,"time":1616716176471,"msg":"Unable to connect to - undefined:undefined (Error: All sentinels are unreachable. Retrying from scratch after 100ms Last error: Connection is closed.)","v":1}`
Any tip how to get Orient Me running?

**Comment by jmosen** [comment, marked as answer]
I solved the problem by restarting the Orient Me deployment with the following command:
` kubectl rollout restart deployment orient-web-client -n connections`

**Reply by dejo1307** [reply]
Sorry for the late response.

Just a quick info on this - this is unfortunately known issue, and somewhere on backlogs. Easiest way to reproduce this is just to reboot the machine where K8s is and you will reproduce it. The reason is the order in which pods get started (yup, shouldn't be like that). But if it happens again, if restarting OM doesn't help, then restart first graphql pod, give it a minute, and then restart OM pod. The sentinels it is talking about are Redis Sentinel pods (which are forwarding requests to Redis pods but in front of them is haproxy pod since you need somehow to balance talking to them).
